docs/reference/snapshot-restore/take-snapshot.asciidoc

```diff
@@ -64,31 +64,57 @@ metadata or its other backing indices.
 [discrete]
 [[create-snapshot-process-details]]
 === Snapshot process details
-The snapshot process is incremental. In the process of making the snapshot, {es} analyses
-the list of the data stream and index files that are already stored in the repository and copies only files that were created or
-changed since the last snapshot. This process allows multiple snapshots to be preserved in the repository in a compact form.
-
-The snapshot process is executed in non-blocking fashion. All indexing and searching operations can continue to run against the data stream or index
-that is being snapshotted. However, a snapshot represents a point-in-time view
-at the moment when snapshot was created, so no records that were added to the data stream or index after the snapshot process was started
-will be included in the snapshot.
-
-The snapshot process starts immediately for the primary shards that have been started and are not relocating at the moment. {es} waits for
-relocation or initialization of shards to complete before snapshotting them.
-
-Besides creating a copy of each data stream and index, the snapshot process can also store global cluster metadata, which includes persistent
-cluster settings, templates, and data stored in system indices, such as Watches and task records, regardless of whether those system
-indices are named in the `indices` section of the request. The <<create-snapshot-api-feature-states,`feature_states` field>> can be used to
-select a subset of system indices to be included in the snapshot. The transient settings and registered snapshot repositories are not stored
-as part of the snapshot.
-
-While a snapshot of a particular shard is being
-created, this shard cannot be moved to another node, which can interfere with rebalancing and allocation
-filtering. {es} can only move a shard to another node (according to the current allocation
-filtering settings and rebalancing algorithm) after the snapshot process
-is finished.
-
-After a snapshot is created, use the <<get-snapshot-api,Get snapshot API>> to retrieve information about a snapshot. See <<snapshots-monitor-snapshot-restore,Monitor snapshot and restore progress>> to learn more about retrieving snapshot status.
+
+The snapshot process works by taking a byte-for-byte copy of the files that
+make up each index or data stream and placing these copies in the repository.
+These files are mostly written by Lucene and contain a compact representation
+of all the data in each index or data stream in a form that is designed to be
+searched efficiently. This means that when you restore an index or data stream
+from a snapshot there is no need to rebuild these search-focussed data
+structures, and also means that you can <<searchable-snapshots,search data in
+the repository without restoring it first>>.
```

Comment:
We may want to explicitly mention searchable snapshots or mounting a snapshot here. The current text implies that the user can run a search on their entire repository, which isn't exactly the case.

Reply:
:+1: see bf7243c93c1b327782744a408d359f42b101dd71.